docs/preview/connected-registry/connected-registry-error-codes.md

```diff
@@ -107,7 +107,7 @@ Sample status detail:
     {
       "code": "DiskError",
       "correlationId": "73a46395-b89b-49c7-5621-d54e8b1574b5",
-      "description": "There is not enough space on the disk.",
+      "description": "No space left on device.",
```

Comment:
Shouldn't this read "Insufficient disk space on the device"?

Reply:
@toddysm "No space left on device" is the standard Linux error message when a file tries to be created when the device is out of disk space